tests/test_estimate_params.py

```diff
@@ -944,8 +944,8 @@ def test_tolerance(self):
 
         # When tolerance is empty list
         m.parameters['thrower_height'] = 3.1
-        m.parameters['throwing_speed'] = 29
-        m.parameters['g'] = 10
+        # m.parameters['throwing_speed'] = 29
+        # m.parameters['g'] = 10
```

Comment:
What was the reason for commenting these out? Should we just delete them?

Reply:
@aqitya outstanding comment that needs addressing

Reply:
Yeah, this needs to be reverted back. I was using some of the previous tests to review my understanding of tolerance. I'll revert them back.